reference/mongodb/bson/packedarray/get.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
+
+<refentry xml:id="mongodb-bson-packedarray.get" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>MongoDB\BSON\PackedArray::get</refname>
+  <refpurpose>Returns the value of an index in the array</refpurpose>
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <modifier>final</modifier> <modifier>public</modifier> <type>MongoDB\BSON\Value</type><methodname>MongoDB\BSON\PackedArray::get</methodname>
+   <methodparam><type>int</type><parameter>key</parameter></methodparam>
+  </methodsynopsis>
+ </refsect1>
+
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  <variablelist>
+   <varlistentry>
+    <term><parameter>key</parameter> (<type>int</type>)</term>
+    <listitem>
+     <para>
+      The index to retrieve from the array
```

Comment:
```suggestion
      The index to retrieve from the array.
```

Parameter descriptions typically end with a period. Feel free to double-check in case I missed others.

Reply:
Went through and fixed a couple more occurrences in the new docs.